### PR TITLE
V3 runtime caching docs

### PR DESCRIPTION
**Pull request by gauntface**
R: @jeffposnick @philipwalton 

Minor tweaks to docs

### CODE REVIEW

File: packages/workbox-runtime-caching/CacheOnly.mjs [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
 
 /**
- * An implementation of a [cache-only](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#cache-only)
+ * An implementation of a
+ * [cache-only]{@link https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#cache-only}
  * request strategy.
  *
- * This class is useful if you want to take advantage of Workbox cache naming
- * and / or plugins.
+ * This class is useful if you want to take advantage of the Workbox plugins.
```
Comment (addyosmani) [inline packages/workbox-runtime-caching/CacheOnly.mjs]: It reads like the previous text here (cache naming/or plugins) was more specific. Was there a reason we wanted the wording more general?

Comment (gauntface) [reply packages/workbox-runtime-caching/CacheOnly.mjs]: IMO the cache naming wasnt a sell to be honest. its trivial to access now.

Comment (gauntface) [reply packages/workbox-runtime-caching/CacheOnly.mjs]: @jeffposnick @philipwalton do you have a preference here?

Comment (jeffposnick) [reply packages/workbox-runtime-caching/CacheOnly.mjs]: Leaving out mention of the cache names is fine. I'm 👍 with the text in your PR.

**Pull request merged**